_platforms/aide.md

```diff
@@ -4,13 +4,10 @@ platform-name: AIDE
 platform-image: /assets/img/platforms/aide_logo.png
 ---
 
-AIDE, the AI Deployment Engine, is an intelligent tool that allows healthcare providers to deploy AI models safely, 
-effectively, and efficiently by enabling the integration of AI models into clinical workflows.
+The AI Deployment Engine (AIDE) is an open-source tool which allows healthcare providers to deploy AI models safely, effectively, and efficiently by enabling the integration of AI models into clinical workflows.  
```

Comment:
AIDE is stylised as AIDE. Like how we call XNAT, XNAT and not extensible neuroimaging archive toolkit (XNAT)